server/services/caseloadService.test.ts

```diff
@@ -31,6 +30,9 @@ describe('Caseload Service', () => {
     prisonCaseload: ['p1', 'p2'],
   } as User
 
+  const getLicencesForOmu = jest.spyOn(licenceService, 'getLicencesForOmu')
+  const getLicencesByNomisIdsAndStatus = jest.spyOn(licenceService, 'getLicencesByNomisIdsAndStatus')
```

Comment:
probably also want to remove the `as jest.Mocked<LicenceService>` if it's no longer a mock

Reply:
My reason for mocking out specific methods was because I wanted to include the tests that determine licence type in these specs. This is a method that's in the licence service, so I couldn't mock out the whole thing.
The method I didn't want to mock is static, but I suppose the licenceService object I'm creating isn't a mock anymore, so I'm happy to remove `as jest.Mocked<LicenceService>` if that's the route we end up with.

I know typically unit testing should only test the specific method you're interacting with, but due to the nature of determining LicenceType (in that it's not only very important to get correct, but also the real-world data going into it is far more complex than small unit tests), I thought utilising the existing setup for the caseloadService specs (which is a reasonable analogue to real-world) to also provide a second-line-of-testing against `getLicenceType` was reasonable.

I do agree that it can cause confusion about why your test is suddenly failing, and perhaps I'm trying to use the caseloadService specs as too much of a mini-integration test, I just thought I'd hijack it while I was there!

I'm happy to put the stubbing back in and remove all of the licenceType checks from the caseloadService specs (stubbing LicenceService will mean that `undefined` is always returned) if you think it's a better approach. I'm hesitant to get into refactoring caseloadService in this PR though as they're largely unrelated to the topic of the changes. I can definitely put it in it's own ticket though.